ublox-bluetooth-commands.go

```diff
@@ -47,7 +48,22 @@ func (ub *UbloxBluetooth) getSerialNumber() (string, error) {
 
 	lines := bytes.Split(sn, []byte("\r\n"))
 	if len(lines) > 0 {
-		return string(lines[0]), nil
+		return strings.Trim(string(lines[0]), "\""), nil
+	}
+
+	return "Unknown", nil
+}
+
+// getLocalName retrieves the local name of the dongle (which will be set to the 8power 16 digit serial number)
+func (ub *UbloxBluetooth) getLocalName() (string, error) {
+	sn, err := ub.writeAndWait(GetLocalName(), true)
```

Comment:
This command is timing out. I am pretty sure the response string is correct but it does not detect the response returned correctly